litestar/contrib/pydantic/pydantic_schema_plugin.py

```diff
@@ -62,73 +66,122 @@
+                        format=OpenAPIFormat.IPV6,
+                        description="IPv6 network",
+                    ),
+                ]
             ),
-            pydantic.UUID4: Schema(
-                type=OpenAPIType.STRING,
-                format=OpenAPIFormat.UUID,
-                description="UUID4 string",
-            ),
-            pydantic.UUID5: Schema(
-                type=OpenAPIType.STRING,
-                format=OpenAPIFormat.UUID,
-                description="UUID5 string",
+            pydantic_v1.Json: Schema(type=OpenAPIType.OBJECT, format=OpenAPIFormat.JSON_POINTER),
+            pydantic_v1.NameEmail: Schema(
+                type=OpenAPIType.STRING, format=OpenAPIFormat.EMAIL, description="Name and email"
             ),
-            pydantic.DirectoryPath: Schema(type=OpenAPIType.STRING, format=OpenAPIFormat.URI_REFERENCE),
-            pydantic.AnyUrl: Schema(type=OpenAPIType.STRING, format=OpenAPIFormat.URL),
-            pydantic.AnyHttpUrl: Schema(
-                type=OpenAPIType.STRING, format=OpenAPIFormat.URL, description="must be a valid HTTP based URL"
-            ),
-            pydantic.FilePath: Schema(type=OpenAPIType.STRING, format=OpenAPIFormat.URI_REFERENCE),
-            pydantic.HttpUrl: Schema(
-                type=OpenAPIType.STRING,
-                format=OpenAPIFormat.URL,
-                description="must be a valid HTTP based URL",
-                max_length=2083,
-            ),
-            pydantic.RedisDsn: Schema(type=OpenAPIType.STRING, format=OpenAPIFormat.URI, description="redis DSN"),
-            pydantic.PostgresDsn: Schema(type=OpenAPIType.STRING, format=OpenAPIFormat.URI, description="postgres DSN"),
-            pydantic.SecretBytes: Schema(type=OpenAPIType.STRING),
-            pydantic.SecretStr: Schema(type=OpenAPIType.STRING),
-            pydantic.StrictBool: Schema(type=OpenAPIType.BOOLEAN),
-            pydantic.StrictBytes: Schema(type=OpenAPIType.STRING),
-            pydantic.StrictFloat: Schema(type=OpenAPIType.NUMBER),
-            pydantic.StrictInt: Schema(type=OpenAPIType.INTEGER),
-            pydantic.StrictStr: Schema(type=OpenAPIType.STRING),
-            pydantic.NegativeFloat: Schema(type=OpenAPIType.NUMBER, exclusive_maximum=0.0),
-            pydantic.NegativeInt: Schema(type=OpenAPIType.INTEGER, exclusive_maximum=0),
-            pydantic.NonNegativeInt: Schema(type=OpenAPIType.INTEGER, minimum=0),
-            pydantic.NonPositiveFloat: Schema(type=OpenAPIType.NUMBER, maximum=0.0),
-            pydantic.PaymentCardNumber: Schema(type=OpenAPIType.STRING, min_length=12, max_length=19),
-            pydantic.PositiveFloat: Schema(type=OpenAPIType.NUMBER, exclusive_minimum=0.0),
-            pydantic.PositiveInt: Schema(type=OpenAPIType.INTEGER, exclusive_minimum=0),
         }
     )
 
-_supported_types = (pydantic.BaseModel, *list(PYDANTIC_TYPE_MAP.keys()))
+
+_supported_types = (pydantic_v1.BaseModel, *list(PYDANTIC_TYPE_MAP.keys()))
```

Comment:
```suggestion
_supported_types = (pydantic_v1.BaseModel, *(k for k in PYDANTIC_TYPE_MAP.keys()))
```
Pedantic, but no need to build the list. Could also be `*iter(PYDANTIC_TYPE_MAP.keys())`.